### PR TITLE
GUI-2332: CloudWatch charts for Auto Scaling metrics on Scaling Group Monitoring page

### PR DESCRIPTION
**Pull request by kamalgill**
Implements https://eucalyptus.atlassian.net/browse/GUI-2332

### CODE REVIEW

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1394/
Test PASSed.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1395/
Test FAILed.

**Comment by aliceh** [comment]
Was changing out GUI worker, sorry.

**Comment by aliceh** [comment]
Retest this, please.

**Comment by euca-qa-gui-tester** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://jenkins.qa1.eucalyptus-systems.com/job/user-console-PR-monitor/1396/
Test PASSed.

File: eucaconsole/static/js/pages/cloudwatch_charts.js [hunk 1]
```diff
@@ -91,7 +91,7 @@ angular.module('CloudWatchCharts', ['EucaConsoleUtils'])
                 yFormat = '.2f';
             }
 
-            if (params.maxValue && params.maxValue < 10) {
+            if (params.maxValue && params.maxValue < 10 && params.unit !== 'Count') {
```
Comment (dkavanagh) [inline eucaconsole/static/js/pages/cloudwatch_charts.js]: Non-intuitive stuff like this could benefit from a 1-line comment.

**Pull request merged**